templates/controller-mutatingwebhookconfiguration.yaml

```diff
@@ -11,25 +11,45 @@ metadata:
     component: controller
```

Comment:
Can we fix the indentation in this file please

Reply:
Hmmm I'm not seeing what's wrong with the indentation here. It's the same number of spaces as on line 10.

Reply:
I commented in the wrong place. But if you look at the file, it adds an extra tab before slice elements (line 41:49 for eg)

Reply:
Oh haha I see. Updated!